### PR TITLE
Fix literal false positives

### LINKED ISSUE #91: False positives: string and number literals
There are some false positives when using literals:

* Stings defined with double-quotes (which is supported according to doc: "Strings may be either single- or double-quoted."):
Code: `<span>[[isAFunction("stringThing!")]]</span>`
Warning: `Property "stringThing!" not found in 'properties' for element 'string-literals'`

* Negative numbers:
Code: `<span>[[isAFunction(-1, myVars)]]</span>`
Warning: `Property -1 not found in 'properties' for element 'number-literals'`

### PR DESCRIPTION
**Pull request by uwolfer**
- string literals defined with double-quotes
- negative numbers

fixes #91

### CODE REVIEW

File: lib/expressions.js [hunk 1]
```diff
@@ -29,7 +29,10 @@ function primaryName(expression) {
   if (expression.name) {
     expression = expression.name;
   }
-  if (expression.indexOf('\'') === 0 || expression.match(/^[0-9]/)) {
+  if (expression.match(/^('|").*('|")$/)) { // string literal
+    return '';
+  }
+  if (expression.match(/^-?\d+/)) { // number literal
```
Comment (rictic) [inline lib/expressions.js]: This regexp will match any expression that starts with a number like `123abc`. Consider this one  instead, which ensures that there's at most one decimal point, and that the expression is entirely numeric.

`/^-?\d+(\.\d+)?$/`

**Comment by rictic** [comment]
One comment, otherwise looks good to me.

I've manually verified that uwolfer has signed the CLA.

File: lib/expressions.js (hunk 1 above)
Comment (uwolfer) [inline lib/expressions.js]: Thanks for your review. I have used a slightly different now, which would also allow things like "-.5". Please note that things like 123e-5 or 0xFF are not supported now (but they are also not supported by Polymer (yet)).

**Comment by rictic** [comment]
LGTM

**Pull request merged**